templates/cli/lib/questions.js.twig

```diff
@@ -212,11 +208,35 @@ const questionsPullProject = [
             }
 
             return choices;
+        },
+        when: (answer) => answer.start === 'existing'
+    }
+];
+
+const questionsPullFunctions = [
+    {
+        type: "checkbox",
+        name: "functions",
+        message: "Which functions would you like to pull?",
+        choices: async () => {
+            const { functions } = await paginate(functionsList, { parseOutput: false }, 100, 'functions');
+
+            return functions.map(func => {
+                return {
+                    name: `${func.name} (${func.$id})`,
+                    value: func
+                }
+            });
         }
+    },
+    {
+        type: "input",
+        name: "override",
+        message: `Are you sure you want to override local functions code and definition? ${chalk.red('All local changes will be lost!')} Type "YES" to confirm.`
```

Comment:
No is also valid answer here, but we make it sound like it's not. Instead of `are you sure` we could ask `do you want to ...`, or something in that tone.